packages/server/src/api/todo/adx/account.ts

```diff
@@ -33,14 +34,17 @@ export default function (server: Server) {
     const cfg = util.getConfig(res)
 
     if (username.startsWith('did:')) {
-      throw new InvalidRequestError(
-        'Cannot register a username that starts with `did:`',
-      )
+      return {
+        status: 400,
+        error: 'InvalidUsername',
+        message: 'Cannot register a username that starts with `did:`',
```

Comment:
they types have this as
```
interface XrpcError {
  name: string
  description?: string
}
```

should this be `description` instead of `message`?

Reply:
actually could we just get the error types in the server's lexicon? like how it's in the clients?

Reply:
You're confusing the Lexicon schema document with the protocol shapes. In the lex schema document, it's `{name, description}` where the description is purely for documentation purposes. The error objects that get returned are `{error, message}`

Reply:
ahhh got it 👍